NodeJS/__integration__/AddressBookClient.it.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  AddressBookClient,
+  createAddressBookClient
+} from '../src/AddressBookClient'
+import { log4js } from '../src/Constants'
+
+const rootLogger = log4js.getLogger()
+rootLogger.level = 'debug'
+rootLogger.debug('Starting AddressBookClient Integration Test')
+
+describe('SubscriptionClient.ts', () => {
+  const localEnv = {
+    user: 'sdetSubAccAdmin',
+    token:
+      'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJzZGV0U3ViQWNjQWRtaW4iLCJpc3MiOiJBQkNfU0VDVVJJVFlfR0FURVdBWSJ9.0HacDx5e1mS9uFmZzrQ8MG8qYp_w27VbsY_z07Tw45w',
+    url: 'https://integration.bpo.avaya.com',
+    subAccountAppId: 'SDE_SDBRec'
+  }
+
+  const { subAccountAppId, token, url } = localEnv
+
+  let addressBookClient: AddressBookClient
+  beforeEach(async () => {
+    addressBookClient = await createAddressBookClient(url, token)
+  })
+
+  test('getAddressBook should return AddressBookPayload with active contacts', async () => {
+    const addressBookPayload = await addressBookClient.getAddressBook()
+    console.log('GET addressBook ', addressBookPayload)
+    expect(addressBookPayload).toBeDefined()
+    expect(addressBookPayload.addressBookId).toBeDefined()
+    expect(addressBookPayload.subAccountAppId).toEqual(subAccountAppId)
+    expect(addressBookPayload.contacts.length).toBeGreaterThan(0)
+  }, 3000)
+
+  test('searchContacts w/o query params should return all active contacts', async () => {
```

Comment:
This isn't a convincing test. Based off of your actual test code, I recommend rewording it like this:
```suggestion
  test('method `searchContacts` does not require parameters', async () => {
```

The wording is a little tricky, but unless you want to make a call with AND without query params to show the difference between the two results, all you can say that a result is returned; you _cannot_ say that "all" contacts are returned unless the code proves that.

Reply:
I see your point. agree

Reply:
renamed